invenio_exporter/views.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-exporter is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio module that adds Exporter programmatic API."""
+
+# TODO: This is an example file. Remove it if you do not need it, including
+# the templates and static folders as well as the test case.
+
+from flask import Blueprint, render_template
+from flask_babelex import gettext as _
+
+blueprint = Blueprint(
+    'invenio_exporter',
+    __name__,
+    template_folder='templates',
+    static_folder='static',
+)
+
+
+@blueprint.route("/")
+def index():
+    """Render a basic view."""
+    return render_template(
+        "invenio_exporter/index.html",
+        module_name=_('invenio-exporter'))
```

Comment:
minor: depends on above, but if no templates it can be removed. In any case `/` route is probably overlapping something.